docs/builtin_elements.md

```diff
@@ -831,6 +831,11 @@ against the constants below.
 * **`Escape`**
 * **`Backtab`**
 * **`Delete`**
+* **`Shift`**
+* **`Control`**
+* **`Alt`**
+* **`AltGr`**
+* **`CapsLock`**
```

Comment:
This is not in sync with the keys. It is missing Meta for example.

(Ideally, this could be auto generated, like the enum docs. But not in this PR because it is already way too big)

Reply:
Well spotted, thanks!